plover_build_utils/testing/steno.py

```diff
@@ -0,0 +1,39 @@
+from plover import system
+from plover.steno import Stroke
+
+
+def steno_to_stroke(steno):
+    if steno_to_stroke.system != system.NAME:
```

Comment:
maybe add a comment explaining why we have to process steno if this condition evaluates as true?